pkg/reconciler/tenancy/workspace/workspace_reconcile_scheduling_test.go

```diff
@@ -470,6 +470,7 @@ func wellKnownClusterWorkspaceTypes() []*tenancyv1alpha1.ClusterWorkspaceType {
 		Name: "universal",
 		Path: "root",
 	}
+	type1.Spec.Initializer = true
```

Comment:
@sttts  shouldn't this `Initializer` be resolved by `r.transitiveTypeResolver.Resolve` ?

Reply:
The initializer name is derived from the CWT workspace and name afaik. It's not resolved.

Reply:
okay but it will be resolved only for the immediate CWT not for all CWTs in the hierarchy, is that okay?

this is how the resolver is wired atm
```
transitiveTypeResolver: clusterworkspacetypeexists.NewTransitiveTypeResolver(func(clusterName logicalcluster.Name, name string) (*tenancyv1alpha1.ClusterWorkspaceType, error) {
				return c.clusterWorkspaceTypeLister.Cluster(clusterName).Get(name)
			}),
```

Reply:
unless there is something wrong with the way the test prepares the data

Reply:
yes, it was a test issue, fixed, PTAL.